### ISSUE TITLE
How can i help with this client?

### ISSUE THREAD

**Issue by ex0rpass** [Wurst-Imperium/Wurst-Client#2233]
I wanna help with this client. and i saw a post by a person on the forum and i want to obfuscation a client called huzuni how do i do it?

**Comment by Alexander01998** [comment]
I don't understand what you are talking about. Please post a link to the forum that you mentioned.

**Comment by ex0rpass** [comment]
@Alexander01998  
Go to #2235 

also im talking about how do i take a hack client code and move it to this client?

**Comment by Alexander01998** [comment]
> also im talking about how do i take a hack client code and move it to this client?

TL;DR: Things don't work like that.

You can't simply take mods from other clients and move them to Wurst, because a) different clients handle their mods differently and are usually incompatible with each other and b) many clients prohibit using their code or place restrictions. I definitely don't want to get into trouble because you violated someone else's license or terms of use.

You can, however, re-invent features from other clients. That means creating a feature yourself that is identical or similar to a feature in another client. For example, nearly every client out there has a Killaura mod, and there surely was a first client to have it. But because all of these clients created their Killaura themselves, it doesn't violate any rights of the creator of the first client that ever had a Killaura.
